index.html

```diff
@@ -147,45 +197,53 @@ <h1 class="aboutmeh1">
-                <p>Frameworks</p>
-            </section>
-            <ul class="aboutmyselflist">
-                <li>React.js</li>   
-                <li>Ruby on Rails</li>
-                <li>RSpec</li>
-                <li>Capybara</li>
-           </ul>
-           <section class="littlecont">
-                <img class="normalimg" src="img/Circle.png" alt="Diamond">
-                <p>Skills</p>
+            <span class="line2"></span>
+            <section class="stats">
+                <div class="statcont">
+                    <section class="littlecont">
+                        <img class="specialimg" src="img/Diamond.png" alt="Diamond">
+                        <p>Language</p>
+                    </section>
+                    <ul class="aboutmyselflist">
+                        <li>Javascript</li>   
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="statcont">
+                    <section class="littlecont">
+                        <img class="normalimg" src="img/Rectangle.png" alt="Diamond">
+                        <p>Frameworks</p>
+                    </section>
+                    <ul class="aboutmyselflist">
+                        <li>React.js</li>   
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                    </ul>
+                </div>
+                <div class="statcont">
+                    <section class="littlecont">
+                        <img class="normalimg" src="img/Circle.png" alt="Diamond">
+                        <p>Skills</p>
+                    </section>
+                    <ul class="aboutmyselflist">
+                        <li>Database Management</li>   
+                        <li>Version Control</li>
+                        <li>CLI</li>
+                        <li>API Design</li>
+                        <li>Web Development</li>
+                    </ul>
+                </div>
             </section>
-            <ul class="aboutmyselflist">
-                <li>Database Management</li>   
-                <li>Version Control</li>
-                <li>CLI</li>
-                <li>API Design</li>
-                <li>Web Development</li>
-            </ul>
         </section>
         <section class="contactsection">
-            <div>
+            <!-- <div class="form"> -->
                 <h1>
                     I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
                 </h1>
-            </div>
+            <!-- </div> -->
             <form action="https://formspree.io/f/xzbqjzwo" class="formlu" method="post">
                 <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required/>
                 <span class="linespan"></span>
```

Comment:
- [ ] Great work on your design, But there are  some aspects you can improve on:
 - Kindly reduce the spacing between the form inputs to match the original design.
 - Adjust the padding of around the placeholder text of the `<textarea>` component to match the original design.
 - Adjust the padding around the `get in touch` button to match the original design
 
![image](https://user-images.githubusercontent.com/98453979/229173689-1ca432a2-686b-4af7-8310-7484f6e686e0.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/229173815-59718e4a-5e6f-4191-ac59-cdcc5446b752.png)